builtin-gconf.c

```diff
@@ -1038,6 +1040,34 @@ gconf_entry_get_value(const GConfEntry *entry)
   return entry ? entry->value : 0;
 }
 
+/** Get reset default value for config entry
+ */
+static const char *gconf_entry_get_default(const GConfEntry *entry)
+{
+  /* Brightness values are a special because the config
+   * defaults use legacy 1-5 range and are migrated to
```

Comment:
Whether the migration has happened is deduced from the values. On startup all values are either in legacy/current state. During reset they change one by one. But I'll check if there is some way to deal with it without causing ringing on dbus / misfiring re-migration etc.